improver/calibration/ensemble_calibration.py

```diff
@@ -729,21 +811,25 @@ def compute_initial_guess(
         Returns:
             list of float:
                 List of coefficients to be used as initial guess.
-                Order of coefficients is [gamma, delta, alpha, beta].
+                Order of coefficients is [alpha, beta, gamma, delta].
 
         """
         if (
             predictor.lower() == "mean"
             and not estimate_coefficients_from_linear_model_flag
         ):
             initial_guess = [0, 1, 0, 1]
-        elif (
-            predictor.lower() == "realizations"
-            and not estimate_coefficients_from_linear_model_flag
+        elif predictor.lower() == "realizations" and (
+            not estimate_coefficients_from_linear_model_flag
+            or not self.statsmodels_found
         ):
-            initial_guess = [0, 1, 0] + np.repeat(
-                np.sqrt(1.0 / no_of_realizations), no_of_realizations
-            ).tolist()
+            initial_guess = (
+                [0]
+                + np.repeat(
+                    np.sqrt(1.0 / no_of_realizations), no_of_realizations
+                ).tolist()
+                + [0, 1]
+            )
```

Comment:
OK while I know this is Black's fault and not yours, this is horrible.  Can I suggest making this a bit clearer by eg:
```
initial_beta = np.repeat(
    np.sqrt(1.0 / no_of_realizations), no_of_realizations
).tolist()
initial_guess = [0] + initial_beta + [0, 1]
```

Reply:
I've modified this formatting as suggested.